File: 1332.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int test, i;
    char str[1000];
    cin >> test;
    for (i = 0; i < test; i++)
    {
        cin >> str;
        int len = strlen(str);
        if (len == 5)
        {
            cout << "3" << endl;
        }
        else if (str[0] == 't' && str[1] == 'w' || str[0] == 't' && str[2] == 'o' || str[1] == 'w' && str[2] == 'o')
        {
            cout << "2" << endl;
        }
        else
        {
            cout << "1" << endl;
        }
    }
}

File: 1332.py
test = int(input())
for i in range(test):
    str = input()
    if len(str) == 5:
        print(3)
    elif(str[0] == 't' and str[1] == 'w' or str[0] == 't' and str[2] == 'o' or str[1] == 'w' and str[2] == 'o'):
        print(2)
    else:
        print(1)
